.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,127 @@
+        run: |
+          echo STEP_ID=${{ matrix.platform }}-${{ matrix.puppet_version }}-1 >> $GITHUB_ENV
+          echo STEP_START=$(date +%s) >> $GITHUB_ENV
+      - name: Checkout Source
+        uses: actions/checkout@v2
+
+      - name: Activate Ruby 2.7
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "2.7"
+          bundler-cache: true
+
+      - name: Print bundle environment
+        run: |
+          echo ::group::bundler environment
+          buildevents cmd $TRACE_ID $STEP_ID 'bundle env' -- bundle env
+          echo ::endgroup::
+      - name: Create the fixtures directory
+        run: |
+          echo ::group::Create the fixtures directory
+          buildevents cmd $TRACE_ID $STEP_ID 'bundle exec rake spec_prep' -- bundle exec rake spec_prep
+          echo ::endgroup::
+      - name: "Honeycomb: Record Setup Environment time"
+        if: ${{ always() }}
+        run: |
+          buildevents step $TRACE_ID $STEP_ID $STEP_START 'Setup Environment'
+          echo STEP_ID=${{ matrix.platform }}-${{ matrix.puppet_version }}-2 >> $GITHUB_ENV
+          echo STEP_START=$(date +%s) >> $GITHUB_ENV
+      - name: Provision test environment
+        run: |
+          buildevents cmd $TRACE_ID $STEP_ID 'bolt plan run servicenow_reporting_integration::provision_machines' -- bundle exec bolt --modulepath spec/fixtures/modules plan run servicenow_reporting_integration::acceptance::provision_machines using='provision_service' image='${{ matrix.platform }}'
+          echo ::group::=== REQUEST ===
+          cat request.json || true
+          echo
+          echo ::endgroup::
+          echo ::group::=== INVENTORY ===
+          if [ -f 'spec/fixtures/litmus_inventory.yaml' ]; then
+            FILE='spec/fixtures/litmus_inventory.yaml'
+          elif [ -f 'inventory.yaml' ]; then
+            FILE='inventory.yaml'
+          fi
+          sed -e 's/password: .*/password: "[redacted]"/' < $FILE || true
+          echo ::endgroup::
+          echo INVENTORY_PATH=$FILE >> $GITHUB_ENV
+      - name: Install server
+        run: |
+          buildevents cmd $TRACE_ID $STEP_ID 'bolt plan run servicenow_reporting_integration::acceptance::server_setup' -- bundle exec bolt --modulepath spec/fixtures/modules plan run servicenow_reporting_integration::acceptance::server_setup puppet_version='${{ matrix.puppet_version }}' -i ./$INVENTORY_PATH
+
+      - name: Install module
+        run: |
+          buildevents cmd $TRACE_ID $STEP_ID 'rake litmus:install_module' -- bundle exec rake 'litmus:install_module'
+
+      - name: Set up Servicenow instance
+        run: |
+          buildevents cmd $TRACE_ID $STEP_ID 'rake acceptance::setup_servicenow_instance' -- bundle exec rake acceptance:setup_servicenow_instance
+
+      - name: Run event tests
+        run: |
+          buildevents cmd $TRACE_ID $STEP_ID 'rspec spec/acceptance/reporting/event_spec.rb' -- bundle exec rspec spec/acceptance/reporting/event_spec.rb
```

Comment:
Are event and incident testing split out just for visibility? We don't usually split out individual spec files in the test yaml like this because it's easy to add a file to the folder but forget to add it to the yaml and then they never run in CI.

Reply:
These are split out because we want to skip the entire `misc_spec` suite on Cloud CI without skipping them when run locally on pooler. So the run_tests task should still run all three but the Cloud CI runner will ignore `misc_spec` because it is entirely eyaml and credential validator tests which won't run on Cloud CI because our cert validation difficulties.